os_net_config/sriov_config.py

```diff
@@ -170,6 +170,9 @@ def get_numvfs(ifname):
     :returns: int -- the number of current VFs on ifname
     :raises: SRIOVNumvfsException
     """
+    cmd = ["/usr/bin/udevadm", "wait", common.get_dev_path(ifname)]
+    logger.debug(f"{ifname}: Running command: {cmd}")
+    processutils.execute(*cmd)
```

Comment:
Please add the processutils.execute under a try except block

Reply:
We can add more readable error messages

Reply:
It will be plenty readable:

```
Timed out for waiting devices being initialized.
```

It depends whether you want to crash or continue processing other vfs.